spec/toplevelmenu/checklist.txt

```diff
@@ -0,0 +1,59 @@
+Description block & feature	Result
+
+TopLevelMenu	////////////
+"Verify that user can hover on the available
+tabs on the TopLevelMenu"	Pass
+"Verify that user can click on the available
+tabs on the TopLevelMenu"	Pass
+"Verify that user can click on the subtabs 
+on the Women tab"	Pass
+"Verify that user can click on the subtabs 
+on the Dresses tab"	Pass
+
+Sidebar (Filter)	////////////
+Verify that 'Tops' filter tab is opens	Pass
+Verify that 'Dresses' filter tab is open	Pass
+Verify that 'Tops' filter tab select	Pass
+Verify that 'Dresses' tab filter select	Pass
+Verify that 'Size' filter tab is select	Pass
+Verify that 'Color' filter tab is select	Pass
+Verify that 'Compositions' filter tab is select	Pass
+Verify that 'Styles' filter tab is select	Pass
+Verify that 'Properties' filter tab is select	Pass
+Verify that 'Availability' filter tab is select	Pass
+Verify that 'Manufacturer' filter tab is select	Pass
+Verify that 'Condition' filter tab is select	Pass
+ Verify that user can edit 'Price' by range filter	Pass
+Verify that user can sort by dropdown list	Pass
```

Comment:
Already reviewed in https://github.com/AndriiMish/TAQC/pull/4/files
Which one PR should be left?